wbooks/tsconfig.json

```diff
@@ -57,7 +57,17 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
     "paths": {
       "@app/*": ["src/app/*"],
-      "@constants/*": ["src/constants/*"]
+      "@constants/*": ["src/constants/*"],
+      "@assets/*": ["./src/app/assets"],
+      "@components/*": ["./src/app/components"],
+      "@hooks/*": ["./src/app/hooks"],
+      "@screens/*": ["./src/app/screens"],
+      "@config/*": ["./src/config"],
+      "@interfaces/*": ["./src/interfaces"],
+      "@redux/*": ["./src/redux"],
+      "@services/*": ["./src/services"],
+      "@utils/*": ["./src/utils"],
```

Comment:
You forgot the `/*` at the end of the lines

Reply:
Fixed